Survey/app/src/main/java/com/example/feverfinder/questions/TextQuestion.java

```diff
@@ -6,21 +6,25 @@
 import android.support.design.widget.TextInputLayout;
 import android.text.Editable;
 import android.text.TextWatcher;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.EditText;
-import android.widget.TextView;
 
 import com.example.feverfinder.R;
 
 public class TextQuestion extends Question implements TextWatcher {
     private String content;
+
     public TextQuestion(String name, String label) {
         super(name, label);
-        content = "";
     }
 
+    @Override
+    public Object getJSONOutput() {
+        return content;
+    }
 
     //TODO: don't want these requires APIs
```

Comment:
Do we know what we're doing about this? (I know it's nothing to do with this PR though but want to note this...)

Reply:
I don't even know what that means. Does it bother us?